.github/workflows/build.yml

```diff
@@ -8,6 +8,11 @@ on:
 
 permissions: read-all
 
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_PROFILE_DEV_DEBUG: 0
+  CARGO_PROFILE_TEST_DEBUG: 0
+
```

Comment:
Does it mean we'll get less information if the test panicked?

If the test failure isn't reproducible, detailed backtrace would be important.

Reply:
Oh, good point. Reverted.